### PR TITLE
Ft/qualitat

### PR DESCRIPTION
**Pull request by NelHim**

### CODE REVIEW

File: src/sections/qualitat.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            </section>
+        </div>
+        
+        <div>
+            <section class="flex flex-col w-full">
+            <div class="flex justify-between px-3 gap-28">
+                <span class=" font-railwayBold">Kommunikations- und Teamfähigkeit</span>
+                <span class="text-xl font-railwayBold">+</span>
+            </div>
+            <span class="h-1 w-full bg-green"></span>
```
Comment (ishimwe07) [inline src/sections/qualitat.html:33]: May be you can try to make those green lines equal as it is in the figma design.

File: src/sections/qualitat.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            </section>
+        </div>
+    </div>
+
+    <p class="text-white lg:pt-28 text-sm font-railwayRegular">Mehr dazu von Matthias Moosburger, Senior Lead IT Architect bei MaibornWolff, <br> der das Ausbildungsprogramm maßgeblich entwickelt hat:</p>
+
```
Comment (ishimwe07) [inline src/sections/qualitat.html:39]: Behind the name "Matthias Moosburger" in that paragraph there is supposed to be a bit rotated green line. So you can consider adding it.

**Review by ishimwe07 — CHANGES_REQUESTED** [review]
You did  a good job, congratulations on that.
But I suggested some changes to make it even better.

**Review by ishimwe07 — APPROVED** [review]
Good.

**Review by divinecharlotte — APPROVED** [review]
# Hello @NelHim      😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 

 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

**Pull request merged**